scala-core-fp/src/test/scala/com/baeldung/scala/monoid/MapMonoidTest.scala

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.scala.monoid
+
+import org.scalatest._
+
+class MapMonoidTest extends WordSpec {
```

Comment:
The unit test files and class names should end in "UnitTest" as per guidelines.

Reply:
@KevinGilmore fixed.